.github/workflows/ci_tests.yaml

```diff
@@ -135,6 +135,7 @@ jobs:
             pytest-mpl
             pytest-rerunfailures
             pytest-xdist
+            libgdal-hdf5
```

Comment:
Remove it after https://github.com/conda-forge/gmt-feedstock/pull/293 is merged.
```suggestion
```